src/linear_solver_logs/solve_log_ma.jl

```diff
@@ -52,18 +70,25 @@ end
     LSLogMA <: LinSysSolverLog
 
 A mutable structure that stores information about a randomized linear solver's behavior.
-    The log assumes that only a random block of full linear system is available for processing. 
-    The goal of this log is usually for all use cases as it acts as a cheap approximation of the 
-    full residual.
+The log assumes that only a random block of full linear system is available for processing. 
+The goal of this log is usually for all use cases as it acts as a cheap approximation of the 
+full residual.
+
+For more information see:
+Pritchard, Nathaniel, and Patel, Vivak. Solving, tracking and stopping streaming linear inverse problems. 
+Inverse Problems 40.8 Web. doi:10.1088/1361-6420/ad5583.
+
+Pritchard, Nathaniel and Vivak Patel. “Towards Practical Large-Scale Randomized Iterative Least Squares 
+Solvers through Uncertainty Quantification.” SIAM/ASA J. Uncertain. Quantification 11 (2022): 996-1024.
```

Comment:
Fix references. Should be same format. Should include DOI if available. Should also be a markdown list.

Reply:
I have updated the reference according to these requests